### PR TITLE
Working UI Prototype

### PR DESCRIPTION
**Pull request by pfrechFra**
First fully Responsive and working Prototype. Please mind the info and readme files.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -5,27 +5,59 @@
   <meta charset="utf-8">
   <title>Application Landscape Viz</title>
   <link rel="stylesheet" href="css/style.css">
-  <script src="libs/gojs-lib/go-debug.js"></script>
+
+  <script src="../gojs-lib/go-debug.js"></script>
```
Comment (antonroesler) [inline public/index.html:9]: That does work. The client cannot exit the public directory. Please locate all library files (eg gojs and go-debug) inside public/libs/...

Comment (antonroesler) [reply public/index.html:9]: This is what it should look like: 
```html
<script src="libs/gojs-lib/go-debug.js"></script>
```

**Review by antonroesler — CHANGES_REQUESTED** [review]
1. Please checkout the comment in index.html regarding the location for libraries 
2. Always before you make a pull request: make sure the code works: Because this code creates a bunch of errors, after fixing the path for go-debug.js there is still one more error:
```error
Failed to load resource: the server responded with a status of 404 (Not Found)
````
Please check that there are no errors

3. I dont think we need to keep the old html / css files you can delete them

**Comment by pfrechFra** [comment]
Rework will be done tomorrow

**Review by fengyilu — APPROVED** [review]

File: public/pics/logo/logo.svg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" standalone="no"?>
```
Comment (leo-hsk) [inline public/pics/logo/logo.svg:1]: Maybe we should change the folder "pics" to "img" it is more common

**Review by leo-hsk — CHANGES_REQUESTED** [review]

**Review by antonroesler — APPROVED** [review]
Supi

**Comment by pfrechFra** [comment]
Design changed and running. Tests passed, please approve to merge

**Review by antonroesler — APPROVED** [review]

**Comment by antonroesler** [comment]
Seems alright, please someone else have a look and approve if u agree with everything.

File: public/css/style.css [hunk 3]
```diff
@@ -44,8 +44,8 @@ body {
 
 
 #diagramDiv {
-  width: 900px; /* Set width only for developing purposes */
-  height: 900px; /* Set width only for developing purposes */
+  width: 980px; /* Set width only for developing purposes */
+  height: 640px; /* Set width only for developing purposes */
```
Comment (antonroesler) [inline public/css/style.css:48]: Ich denke es wäre besser nirgendwo im css px zu benutzten. Immer besser % oder?

Comment (pfrechFra) [reply public/css/style.css:48]: Ich bin generell nicht zufrieden mit der Lösung das wir dem Element eine feste Größe geben müssen. Es scheint aber zwingend für das Diagram zu sein.

File: public/pics/logo/logo.svg (hunk 2 above)
Comment (pfrechFra) [reply public/pics/logo/logo.svg]: changed

**Review by antonroesler — APPROVED** [review]

**Review by fengyilu — APPROVED** [review]

**Pull request merged**